db/view/view.cc

```diff
@@ -332,14 +378,33 @@ static query::partition_slice make_partition_slice(const schema& s) {
             std::move(opts));
 }
 
+view_builder::build_step& view_builder::get_or_create_build_step(utils::UUID base_id) {
+    auto it = _base_to_build_step.find(base_id);
+    if (it == _base_to_build_step.end()) {
+        auto base = _db.find_column_family(base_id).shared_from_this();
```

Comment:
This can potentially return a different base each time. Perhaps a base with different columns, or without the view. How safe are we against that?

Reply:
Github shows as if I created this function, but it's actually just code movement - whatever the issue might be here, it's preexisting. I think that view building gets restarted from scratch if the base table changes, but I'll need to verify it asynchronously to this series.

Reply:
Sure.